js/dlg/message-box.css

```diff
@@ -32,6 +30,11 @@
   -moz-user-select: text;
 }
 
+#message-box > div:not(.dragged) {
```

Comment:
Now that `translate` is used this change can be reverted.

Reply:
Not really... the `top` and `right` property have for some reason still affect over the `translate` property. So it will disappear somewhere far far away not visible on the screen once moving the box. Alternatively an `unset` or `revert` can be set into the inline style for these 2 properties.

Reply:
I've made such dragging code myself so I know it works. The calculation you used for `translate` is wrong. There should be no need for getBoundingClientRect. Just use the difference between the original clicked clientX and the current clientX.

Reply:
Ah Sorry, my bad. I made the calculation on a false assumption of the returned property inside the event.  It should be good now and doesn't rely on getBoundingClientRect or offset. So it the 2 properties(`right` and `top`) doesn't matter anymore for the code.